labeling/labelers.go

```diff
@@ -0,0 +1,42 @@
+package labeling
+
+import (
+	"strings"
+
+	bsky "github.com/bluesky-social/indigo/api/bsky"
+)
+
+// simple record labeling (without pre-fetched blobs)
+type SimplePostLabeler interface {
+	labelPost(p bsky.FeedPost) []string
+}
+type SimpleActorProfileLabeler interface {
+	labelActorProfile(ap bsky.ActorProfile) []string
```

Comment:
Why have an interface with unexported methods?

Reply:
This keyword labeling is kind of throw-away just as an example for testing and integration, but I can expose these methods.

Reply:
made public